### ISSUE TITLE
New issue created don't show up in overview feed

### ISSUE THREAD

**Issue by ehausen** [mirumee/overv.io-beta#512]
If you create a new on github or overv.io it does show a notification, but does not show up in the **overview feed** to the right. If you add labels or assign at the same time when you add, it does show up in the overview but only marked as changes to a new issue. 

So there is no way of telling what issues were created by looking at the feed. I would expect to see the newly created issues in the feed so that I can keep track of what's happening.

**Comment by mirekm** [comment]
There are two types of event feeds at the moment: the one related to the issues you've been assigned to (or mentioned in) and the second one with events from all the available issues. However, the second one is only implemented in the mobile application (Android currently https://play.google.com/store/apps/details?id=io.overv). We will most likely merge these two into a single stream and provide appropriate filtering instead. Can you confirm you're assigned to (or mentioned in) the issue you're referring to?

**Comment by ehausen** [comment]
I'm testing alone right now, so I'm always the creator. Of course I can self assign.. Not sure if that is what you meant.

**Comment by mirekm** [comment]
Sorry if this was not clear enough. Our original goal was to present the relevant events only — namely related to the issues that user really cares about. As long as the meaning of this is discussable and open to further amendments the current browser/desktop application (in the Overview panel) lists only events related to the issues specifically assigned to the user or mentioning him. This possibly makes more sense when you work in a bigger team — then you don't really want to know everything about all the issues (even those you created by yourself but you're not assigned to or mentioned in).
Currently the full stream is only available in our Android application but will be soon also available in a main browser app.

The bottom line is that as long as you're not assigned to the issue and the issue doesn't mention you explicitly with @ you're not meant to see it in the current Overview panel. I hope this makes more sense now.

**Comment by ehausen** [comment]
I understand the logic, yes. :)

Right now just I might have a need to keep an eye on the incoming issues, label and assign them, etc.

PS. Mobile application now installed :sunglasses:
